### ISSUE TITLE
Powershell - How to?

### ISSUE THREAD

**Issue by PHagenfeldt** [micjahn/ZXing.Net#219]
Hi,

Not really an issue but I can't seem to figure out how to get ZXing working in Powershell. 

I'm fairly new to PowerShell and not a developer and have been setting up automations in folders etc. 

Is there any step by step documentation on how to get it working locally on a PC?

I have tried downloading the repo and pointing the path in the script to xzing.dll. PowerShell gives me an error at dispose and says there's a missing ')'.

Would anybody care to help a novice out?

**Comment by micjahn** [comment]
Did you try it with the code snippet from the wiki?
https://github.com/micjahn/ZXing.Net/wiki/Using-ZXing.Net-with-Powershell

**Comment by PHagenfeldt** [comment]
@micjahn Yeah, I copied it and changed the path to point at where I have the dll which I extracted from the nuget file, but not quite sure I downloaded it correctly and that I'm pointing it at the right folder.

Steps:
Downloaded the nuget and extracted the folder
Put in the code snippet from the wiki in PowerShell IDE
Changed "PS C:\>Add-Type -Path <place the path to the zxing.dll here>\zxing.dll" to where I have the content from the nuget package

Am I doing something wrong here? I don't have access to the computer until tomorrow I'm afraid.

**Comment by micjahn** [comment]
Please post a full example of your code and the error message.
I tried the sample from the wiki and it works perfectly.

**Comment by PHagenfeldt** [comment]
@micjahn 

Hi!

The code:
PS C:\>Add-Type -Path C:\Users\redacted\Pictures\xzing\net4.5\zxing.dll
PS C:\>$reader = New-Object -TypeName ZXing.BarcodeReader
PS C:\>$reader.Options.TryHarder = 1 # set TryHarder option to true, other options can be set the same way
PS C:\>$bitmap = [System.Drawing.Bitmap]::FromFile("C:\Users\redacted\Pictures\b_sephilipäger-20191025-13_55")
PS C:\>$result = $reader.Decode($bitmap)
PS C:\>$bitmap.Dispose()
PS C:\>$result.Text

The error:
At line:6 char:24
+ PS C:\>$bitmap.Dispose()
+                        ~
An expression was expected after '('.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : ExpectedExpression

**Comment by PHagenfeldt** [comment]
@micjahn 

Hi!
I realised that I needed to remove PS C:> from my PowerShell script. Works like a charm now!
